udata/core/reuse/api.py

```diff
@@ -101,7 +101,7 @@ class ReuseListAPI(API):
     @api.marshal_with(reuse_page_fields)
     def get(self):
         args = reuse_parser.parse()
-        reuses = Reuse.objects(deleted=None, private__ne=True)
+        reuses = Reuse.objects(archived=None, deleted=None, private__ne=True)
```

Comment:
For now, we can keep returning archived reuses on list and suggest endpoints

Reply:
Thanks, that makes sense. Fixed